tensorboard/compat/proto/op_def.proto

```diff
@@ -175,7 +174,6 @@ message OpDef {
   // trigger TF network failure handling logics.
   bool is_distributed_communication = 21;
 }
-// LINT.ThenChange(
 //     https://www.tensorflow.org/code/tensorboard/compat/proto/op_def_util.cc)
```

Comment:
Kind of a nit, but for cases like this, this leaves behind a somewhat confusing fragment with the latter half of the LINT directive. Would be a bit nicer if we either were to strip the entire thing, or leave some explanatory replacement like `// Disabled ThenChange block: (`